apptail/instance.go

```diff
@@ -37,7 +38,7 @@ func (instance *Instance) Identifier() string {
 }
 
 // Tail begins tailing the files for this instance.
-func (instance *Instance) Tail() {
+func (instance *Instance) Tail(tracker storage.Tracker) {
```

Comment:
Right; I just tend to favour putting things that don't change over time to be composed, but that's more a personal quirk of disliking lots of function call arguments than some wisdom-based thing or anything like that.  Meh, I guess if it works as-is it's fine.